hhvm.nix

```diff
@@ -223,6 +226,17 @@ stdenv.mkDerivation rec {
         third-party/proxygen/bundled_proxygen-prefix/src/bundled_proxygen
     '';
 
+  doCheck = true;
+
+  checkPhase =
+    ''
+      set -ex
+      runHook preCheck
+      export HHVM_BIN="$PWD/hphp/hhvm/hhvm"
+      (cd ${./.} && "$HHVM_BIN" hphp/test/run.php quick)
+      runHook postCheck
+    '';
+
```

Comment:
According to https://nixos.org/manual/nixpkgs/unstable/#ssec-check-phase, `checkPhase` is supposed to used to run test suites of a package.